# python_reproducer_for_segfault
This is a reproducer for a segfault in Python 3.6.7

To reproduce this issue I run the following commands on a newly deployed vm on Debian GNU/Linux 9 (stretch):

'''
sudo apt-get install git -y
git clone https://github.com/pippy360/python_reproducer_for_segfault.git
cd python_reproducer_for_segfault/
sudo apt-get install python3-pip -y
pip3 install .
pip3 install .
'''

output:
'''
'''
